crypto/src/hash/rescue/rp64_256/mod.rs

```diff
@@ -374,349 +412,346 @@ impl Rp64_256 {
 // MDS
 // ================================================================================================
 /// Rescue MDS matrix
-/// Computed using algorithm 4 from <https://eprint.iacr.org/2020/1143.pdf>
```

Comment:
It would probably be good to add a very brief explanation (1 - 2 sentences) about this MDS matrix.